src/riot/WagtailPages/CoursePage.riot.html

```diff
@@ -123,6 +133,38 @@ <h3>{ TRANSLATIONS.download_course() }</h3>
                     })
             },
 
+            onMounted(props, state) {
+                const descEl = this.$('.clamp-3-lines');
+                if (descEl.scrollHeight > descEl.clientHeight) {
+                    this.update({hasExpandableDescription: true})
+                }
```

Comment:
This line caused an issue on jid-staging deploy, i edited locally and published like this
```suggestion
                const descEl = this.$('.clamp-3-lines');
                if (descEl && descEl.scrollHeight > descEl.clientHeight) {
                    this.update({hasExpandableDescription: true})
                }
```